gremlin-go/driver/anonymousTraversal.go

```diff
@@ -19,534 +19,766 @@ under the License.
 
 package gremlingo
```

Comment:
Did you intentionally leave out the '.' for all your comments?

Reply:
Are they supposed to be there? They originally didn't exist on the AnonymousTraversal

Reply:
Not sure, personally IDC, I am pretty sure it does not affect the quality of the comment in any measurable way.